src/main/java/bio/terra/cli/businessobject/Resource.java

```diff
@@ -59,7 +59,7 @@ public enum Type {
     GCS_OBJECT,
     BQ_DATASET,
     BQ_TABLE,
-    AI_NOTEBOOK,
+    GCP_NOTEBOOK,
```

Comment:
Let's not change the enum value in this PR. It will likely break cached resources or scripts that depend on the JSON format.

Reply:
Hi Jay. The reason why I changed all name, because would like to make it consistent. 
or it will have to define the type as AI_NOTEBOOK 
![Screen Shot 2022-06-14 at 10 09 06 AM](https://user-images.githubusercontent.com/100877272/173598059-0dd7434e-9442-48ad-8ca9-c4c8502240a3.png)

Reply:
Talk with Jay. JSON-sensitive value needs to stay the same. 
And will have a comments at the place to show ai_notebook is gcp_notebook.